### DISCUSSION TITLE
Re-authentication is continually triggered

### DISCUSSION THREAD

**Discussion by amdrsantos** [strongswan/strongswan#1030] [Q&A] [answered]
When a server has a re-auth/re-key time extremely high compared to the remote peer, it will cause successive re-auth, because the formulas used:

rekey_time = 4h = 240m
over_time = 0.1 * rekey_time = 24m

If the peer is set such that lifetime is less or equal than 24m then a re-authentication is continually triggered.

Is there a way to prevent this to happen?

**Comment by tobiasbrunner** [comment, marked as answer]
Adjust the values (on client and/or server) or disable reauthentication on the server so it does not request clients to reauthenticate. (Note that `rekey_time` has nothing to do with reauthentication, which is triggered by `reauth_time`, see [here](https://docs.strongswan.org/docs/5.9/config/rekeying.html).)

**Reply by amdrsantos** [reply]
Thanks Tobias, 
Besides changing the configurations and disabling authentication, are there other options for this?

**Reply by tobiasbrunner** [reply]
What do you have in mind?

**Reply by amdrsantos** [reply]
To disallow that IKE-peer from connecting for a certain time, like a back-off time.

**Reply by tobiasbrunner** [reply]
How would that improve the situation? There would still constantly be reauthentications, no?

**Reply by amdrsantos** [reply]
Yes, indeed.

I was testing against a 5.7.2. version:

This seems to not be happening in the last releases:
In https://github.com/strongswan/strongswan/releases/tag/5.9.4 it is stated:
 AUTH_LIFETIME notifies are now only sent by a responder if it can't reauthenticate the IKE_SA itself due to asymmetric authentication (i.e. EAP) or the use of virtual IPs.

Can you confirm that is fixed?

**Reply by tobiasbrunner** [reply]
Depends on your config. But in symmetric scenarios recent versions don't send notifies to request the peers to reauthenticate, but just reauthenticate themselves if configured as such.

**Reply by amdrsantos** [reply]
> Depends on your config. But in symmetric scenarios recent versions don't send notifies to request the peers to reauthenticate, but just reauthenticate themselves if configured as such.

Right. Thanks.
